atef/type_hints.py

```diff
@@ -1,4 +1,6 @@
+import pathlib
 from typing import Union
 
+AnyPath = Union[str, pathlib.Path]
```

Comment:
All of these type hint shortcuts should really be in the core python typing module, in my opinion. They are broadly useful.

Reply:
Totally agreed...